packages/elements/src/utils/completed-section/completed-sections.ts

```diff
@@ -33,32 +30,14 @@ export const isCompletedSecondaryID = (identityCheck?: IdentityCheckModel | null
 
 export const isCompletedAddress = (contact?: ContactModel | null) => {
   const isValidPrimaryAddress =
-    !!contact?.primaryAddress?.buildingName &&
-    !!contact?.primaryAddress?.buildingNumber &&
     !!contact?.primaryAddress?.line1 &&
-    !!contact?.primaryAddress?.line2 &&
     !!contact?.primaryAddress?.line3 &&
-    !!contact?.primaryAddress?.line4 &&
+    !!contact?.primaryAddress?.postcode &&
     !!contact?.metadata?.primaryAddress?.year &&
     !!contact?.metadata?.primaryAddress?.month &&
     !!contact?.metadata?.primaryAddress?.documentType &&
     !!contact?.metadata?.primaryAddress?.documentImage
-  const isValidSecondaryAddress =
```

Comment:
Why do we need to remove this one?

Reply:
Because It is not a requirement to fill in the ‘Less that 3 years’ section in order to make ‘Address History’ complete